pyscf/scf/hf.py

```diff
@@ -1710,6 +1794,15 @@ def init_guess_by_1e(self, mol=None):
         mo_occ = self.get_occ(mo_energy, mo_coeff)
         return self.make_rdm1(mo_coeff, mo_occ)
 
+    @lib.with_doc(init_guess_by_sap.__doc__)
+    def init_guess_by_sap(self, mol=None, sap_basis='sapgraspsmall', **kwargs):
+        if mol is None: mol = self.mol
+        logger.info(self, '''Initial guess from superposition of atomic potentials,
+using procedure laid out by S. Lehtola at 
+doi:10.1021/acs.jctc.8b01089 and
+doi:10.1063/5.0004046.''')
+        return init_guess_by_sap(mol, sap_basis=sap_basis, **kwargs)
```

Comment:
This is where you should build the `sap_basis` if it is not already a PySCF dictionary